src/object/interrupt.c

```diff
@@ -134,8 +134,15 @@ exception_t decodeIRQHandlerInvocation(word_t invLabel, irq_t irq)
 void invokeIRQHandler_AckIRQ(irq_t irq)
 {
 #ifdef CONFIG_ARCH_RISCV
-    return;
+#if !defined(CONFIG_PLAT_SPIKE)
+    /* QEMU has a bug where interrupts must be
```

Comment:
1. Do you know if this apply to all versions of QEMU?
2. This should also apply to `CONFIG_PLAT_QEMU_RISCV_VIRT`

Reply:
@Furao do you know the answer to this? We can update the ifdef easy enough.

Reply:
It's better to use explicit platform checks, so it is clear where this applies. Please check for `CONFIG_PLAT_QEMU_RISCV_VIRT` for now, as this is the official QEMU platform. 
There are other platforms (HiFive, PolarFire) where a QEMU simulation also exists. They may have the same problem, but here we should consider the real hardware as the default target. An additional kernel config define should be added to explicitly enable the fix for this specific QEMU bug.

Reply:
I agree that we should do an explicit check for QEMU_RISCV_VIRT, but that platform doesn't exist in `riscv_he_v0.6`, hence the check for `spike`.

@axel-h @Ivan-Velickovic do you think this PR should be rebased onto master?

Reply:
Ah, my bad. I forgot that this branch wouldn't have support for virt yet. Yes, I think it's a good idea to rebase the branch on master.

Reply:
Just to clarify, you're saying I should rebase `rocketchip_bug_fixes_zcu102` onto master, not `riscv_he_0.6`?

Reply:
It seems @Ivan-Velickovic ist trying to rebase `riscv_he_0.6`, so you can rebease on it then.

Reply:
I rebased this branch onto `master`